docs/tempo/website/cli/_index.md

```diff
@@ -74,6 +74,7 @@ Arguments:
 - `tenant-id` The tenant ID.  Use `single-tenant` for single tenant setups.
 
 Options:
+- `--include-compacted` Include blocks that have been compacted. Default behavior is to display only active blocks.
 - `--load-index` Also load the block indexes and perform integrity checks for duplicates. **Note:** can be intense.
```

Comment:
```suggestion
- `--load-index` Also load the block indexes and perform integrity checks for duplicates. **Note:** This can be a resource intensive process.
```